.github/actions/integration-tests/base

```diff
@@ -31,6 +31,19 @@ function log_rc {
     fi
 }
 
+function validate_ci_build_controller {
+    case "${CI_BUILD_CONTROLLER}" in
+        yes|no)
+            echo "CI Build Controller is '${CI_BUILD_CONTROLLER}'"
+            echo
+            ;;
+        *)
+            echo "ERROR: Unknown CI_BUILD_CONTROLLER value [${CI_BUILD_CONTROLLER}]"
```

Comment:
you could just print a warning and default to no

Reply:
This is just being consistent with some of the other validators that are already present.  They don't make a lot of sense when running the CI via Github runners (self hosted or not) but it is possible to execute the CI manually outside of the Github environment and so parameter validation is important for that use case.